src/graph/commands/epicurve/epicurve.pas

```diff
@@ -0,0 +1,216 @@
+unit epicurve;
```

Comment:
A lot of the comments from barchart also apply here. Especially in relations to command options and potential memory leaks.

Reply:
Yes, I've been fixing them in parallel